tests/test_forecast.py

```diff
@@ -28,6 +30,17 @@ def test_forecast():
         )
         == datetime
     )
+    assert abs(weather_forecast.nearest_forecast["dt"] - now_ts) == min(
+        [abs(x["dt"] - now_ts) for x in weather_forecast.forecast]
+    )
+    assert (
+        current_forecast["dt"] < datetime.utcnow().timestamp()
+        or current_forecast["dt"] == weather_forecast.forecast[0]["dt"]
```

Comment:
With previous comment

```suggestion
        or not current_forecast["dt"]
```